tools/generate_requirements.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Generate requirements/*.txt files from pyproject.toml."""
+
+import sys
+from pathlib import Path
+
+try:  # standard module since Python 3.11
+    import tomllib as toml
+except ImportError:
+    try:  # available for older Python via pip
+        import tomli as toml
+    except ImportError:
+        sys.exit("Please install `tomli` first: `pip install tomli`")
+
+repo_dir = (Path(__file__).parent / "..").resolve()
```

Comment:
```suggestion
repo_dir = Path(__file__).parent.parent
```
I think that gives the same result?

Reply:
Yes it does, thanks!